src/biogeophys/SnowSnicarMod.F90

```diff
@@ -578,73 +808,162 @@ subroutine SNICAR_RT (flg_snw_ice, bounds, num_nourbanc, filter_nourbanc,  &
 
                   ! Pre-emptive error handling: aerosols can reap havoc on these absorptive bands.
                   ! Since extremely high soot concentrations have a negligible effect on these bands, zero them.
-                  if ( (numrad_snw == 5).and.((bnd_idx == 5).or.(bnd_idx == 4)) ) then
+                  if ( (snicar_numrad_snw == 5).and.((bnd_idx == 5).or.(bnd_idx == 4)) ) then
+                     mss_cnc_aer_lcl(:,:) = 0._r8
+                  endif
+
+                  if ( (snicar_numrad_snw == 3).and.(bnd_idx == 3) ) then
```

Comment:
@cenlinhe I think I should remove code mentioning `snicar_numrad_snw == 3`, right?

Reply:
Yes, correct.

Reply:
What about replacing `snicar_numrad_snw <= 5` with `snicar_numrad_snw == 5`? Do you prefer to keep the flexibility of using other values (i.e. not 5 and not 480)?

Reply:
I think we can just use replace snicar_numrad_snw <= 5 with snicar_numrad_snw == 5, since in the next few years, we are probably not going to add more options for using other values.